core-bundle/contao/drivers/DC_Folder.php

```diff
@@ -2497,9 +2492,9 @@ public function ajaxTreeView($strFolder, $level)
 
 		$this->isValid($strFolder);
 
-		if (!is_dir($this->strRootDir . '/' . $strFolder) || !$this->isMounted($strFolder))
+		if (!is_dir($this->strRootDir . '/' . $strFolder) || !$this->isMounted($strFolder) && !\in_array($strFolder, $this->visibleRootTrails, true))
```

Comment:
We should probably add parenthesis to clarify if it is `$a || ($b && $c)` or `($a || $b) && $c`.

Reply:
Done in c31d59f4d1ac2330a66f1dd2e01f8b1e019436c4.